cli/cmd/policy.go

```diff
@@ -590,6 +591,13 @@ func buildPolicyDetailsTable(policy api.Policy) string {
 		{"UPDATED BY", policy.LastUpdateUser},
 		{"EVALUATION FREQUENCY", policy.EvalFrequency},
 	}
+	// append VALID EXCEPTION CONSTRAINTS to the table when the ExceptionConfiguration is not empty
+	exceptionConstraints := strings.Join(
+		getPolicyExceptionConstraintsSlice(policy.ExceptionConfiguration), ", ")
+	if exceptionConstraints != "" {
+		entry := []string{"VALID EXCEPTION CONSTRAINTS", exceptionConstraints}
+		details = append(details, entry)
+	}
```

Comment:
@afiune any thoughts on whether we want to just drop this field from the table, or whether we should put a message  to the effect of `no constraints found`

Reply:
Let's add it always and if the policy do not have constraints then show "None" - because no constraint found implies that maybe, there could be constraints.

Reply:
Sounds good. I'll update!